src/libraries/System.Private.CoreLib/src/System/ReadOnlyMemory.cs

```diff
@@ -358,13 +362,13 @@ public unsafe MemoryHandle Pin()
                     // Array is already pre-pinned
                     if (_index < 0)
                     {
-                        void* pointer = Unsafe.Add<T>(Unsafe.AsPointer(ref Unsafe.As<T[]>(tmpObject).GetRawSzArrayData()), _index & RemoveFlagsBitMask);
+                        void* pointer = Unsafe.Add<T>(Unsafe.AsPointer(ref MemoryMarshal.GetRawArrayData(Unsafe.As<T[]>(tmpObject))), _index & ReadOnlyMemory<T>.RemoveFlagsBitMask);
```

Comment:
Will revert `ReadOnlyMemory<T>.RemoveFlagsBitMask` back to simply `RemoveFlagsBitMask`. Missed this during initial review.